### ISSUE TITLE
[Additional Feature] SLO views (Single Logout)

### ISSUE THREAD

**Issue by peppelinux** [OTA-Insight/djangosaml2idp#23]
We should be able to expose a correct web resource for managing Single Logout requests, in HTTP-POST (SOAP), from SP.

Actually we have only these
````urlpatterns = [
    path('sso/post', views.sso_entry, name="saml_login_post"),
    path('sso/redirect', views.sso_entry, name="saml_login_redirect"),
    path('sso/init', views.SSOInitView.as_view(), name="saml_idp_init"),
    path('login/process/', views.LoginProcessView.as_view(), name='saml_login_process'),
    path('login/process_multi_factor/', views.ProcessMultiFactorView.as_view(), name='saml_multi_factor'),
    path('metadata/', views.metadata, name='saml2_idp_metadata'),
]
````

Could we develop also the following, I apologize that I still have to read some more source code from pysaml2 example folder to get this more defined here.
````
    path('slo/post', views.sso_entry, name="saml_slo_post"),
    path('slo/init', views.SLOInitView.as_view(), name="saml_slo_init"),
````

Any ideas or suggestions?

**Comment by peppelinux** [comment]
Implemented and tested in SLO branch:
https://github.com/peppelinux/djangosaml2idp/tree/SLO

**Comment by peppelinux** [comment]
I'm pushing PR to @askvortson1 repository for faster opinions and coordinati on, this features was pusher in a separate branch. In It i coded lazy gettext and a general refactor - not invasive - of the previous code. I Hope to do it Better in the future

**Comment by mhindery** [comment]
SLO is available in the package v0.6.0.
